src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -190,11 +195,13 @@ public void periodic() {
     tof1Freq = tof1DutyCycleInput.getFrequency();
     tof1DutyCycle = tof1DutyCycleInput.getOutput();
     tof1Range = tof1ScaleFactor * (tof1DutyCycle / tof1Freq - 0.001);
-    SmartDashboard.putNumber("TOF 1/Frequency", tof1Freq);
-    SmartDashboard.putNumber("TOF 1/Duty Cycle", tof1DutyCycle);
-    SmartDashboard.putNumber("TOF 1/Time", tof1DutyCycle / tof1Freq);
     SmartDashboard.putNumber("TOF 1/Range", tof1Range);
-    SmartDashboard.putBoolean("TOF 1/Closed", tof1Range < Constants.kTOF1_closed);
+    if (debug) {
+      SmartDashboard.putNumber("TOF 1/Frequency", tof1Freq);
+      SmartDashboard.putNumber("TOF 1/Duty Cycle", tof1DutyCycle);
+      SmartDashboard.putNumber("TOF 1/Time", tof1DutyCycle / tof1Freq);
+      SmartDashboard.putBoolean("TOF 1/Closed", tof1Range < Constants.kTOF1_closed);
```

Comment:
I think they may want to see TOF X/Closed, not tof1Range normally on the driver's station (but not positive)